Library/Homebrew/test/cleanup_spec.rb

```diff
@@ -104,14 +104,30 @@
     expect(f4).to be_installed
   end
 
-  specify "::cleanup_logs" do
-    path = (HOMEBREW_LOGS/"delete_me")
-    path.mkpath
-    ARGV << "--prune=all"
+  describe "::cleanup_logs" do
+    let(:path) { (HOMEBREW_LOGS/"delete_me") }
 
-    described_class.cleanup_logs
+    before do
+      path.mkpath
+    end
+
+    it "cleans all logs if prune all" do
+      ARGV << "--prune=all"
+      described_class.cleanup_logs
+      expect(path).not_to exist
+    end
 
-    expect(path).not_to exist
+    it "cleans up logs if older than 14 days" do
+      allow_any_instance_of(Pathname).to receive(:mtime).and_return(Time.now - 60 * 60 * 24 * 15)
+      described_class.cleanup_logs
+      expect(path).not_to exist
+    end
+
+    it "does not clean up logs less than 14 days old" do
+      allow_any_instance_of(Pathname).to receive(:mtime).and_return(Time.now - 60 * 60 * 24 * 2)
```

Comment:
May be worth having a `let` for the `Time.now - 60 * 60 * 24 * 2` as it's repeated three times. Also may as well do it for the `Time.now - 60 * 60 * 24 * 15` just to be consistent.

Reply:
I'm doing a `let` for `Time.now - 60 * 60 * 24 ` and call it num_of_days.